defaults/main.yml

```diff
@@ -43,7 +50,12 @@ consul_datacenter: "{{ lookup('env','CONSUL_DATACENTER') | default('dc1', true)
 consul_domain: "{{ lookup('env','CONSUL_DOMAIN') | default('consul', true) }}"
 consul_log_level: "{{ lookup('env','CONSUL_LOG_LEVEL') | default('INFO', true) }}"
 consul_syslog_enable: "{{ lookup('env','CONSUL_SYSLOG_ENABLE') | default(true, true) }}"
-consul_iface: "{{ lookup('env','CONSUL_IFACE') | default(ansible_default_ipv4.interface, true) }}"
+consul_iface: "\
+  {% if ansible_os_family == "Windows" %}\
+    {{ lookup('env','CONSUL_IFACE') | default(ansible_interfaces[0].interface_name, true) }}\
```

Comment:
Network interfaces in Windows are handled very differently in Ansible, with very little overlap. I'm not sure why. But this works.